tools/benchmark.py

```diff
@@ -391,6 +391,20 @@ def start_pipelinerun_thread(body, namespace):
     logging.debug(f"Created PipelineRun: {json.dumps(response)[:100]}...")
 
 
+def fetch_current_concurrency(value):
+    # Check if the value is a number
+    if value.isdigit():
+        return int(value)
+    # Check if the value is a valid file path
+    if os.path.isfile(value):
+        with open(value, "r") as file:
+            contents = file.read().strip()
+        if contents.isdigit():
+            return int(contents)
+        else:
+            logging.error(f"File '{value}' does not contain a valid integer")
```

Comment:
I would make this an exception instead of just logging error - error can be missed, but exception ruins the whole run, so it will be hard to miss I hope :)